src/FNAPlatform/SDL2_FNAPlatform.cs

```diff
@@ -970,6 +1011,10 @@ public static void RunLoop(Game game)
 				}
 
 				Keyboard.SetKeys(keys);
+				if (GetTouchCapabilities().IsConnected)
```

Comment:
This adds an allocation per frame... may just have to check for init, or call SDL_GetNumTouchDevices directly. I'd say check for touch on FNAPlatform.Init but that Windows thing is killing us...